src/components/BookList.js

```diff
@@ -1,21 +1,26 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
 import BookItem from './BookItem';
 
-const books = [
-  {
-    id: 1,
-    title: 'Whenever you can',
-    author: 'Michael Douglas',
-    category: 'Hollywood movies',
-  },
-];
+// const books = [
+//   {
+//     id: 1,
+//     title: 'Whenever you can',
+//     author: 'Michael Douglas',
+//     category: 'Hollywood movies',
+//   },
+// ];
```

Comment:
- [ ] I would recommend removing the commented code as it makes the codebase cleaner.